core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\CoreBundle\DataContainer\DataContainerOperation;
+use Contao\CoreBundle\Security\ContaoCorePermissions;
+use Contao\CoreBundle\Security\DataContainer\CreateAction;
+use Contao\CoreBundle\Security\DataContainer\DeleteAction;
+use Contao\CoreBundle\Security\DataContainer\UpdateAction;
+use Contao\CoreBundle\ServiceAnnotation\Hook;
+use Contao\DataContainer;
+use Doctrine\DBAL\Connection;
+use Symfony\Component\Security\Core\Security;
+
+/**
+ * @internal
+ *
+ * @Hook("loadDataContainer", priority=200)
```

Comment:
Any reason for 200? Usually it's any power of 2 :) So 256 maybe?

Reply:
no reason, just a randomly "high" number 😂